changelog/3567.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Various updates to exceptions to save and restore more properties
+during the pickle/serialization process.
```

Comment:
Please format the message according to https://github.com/urllib3/urllib3/tree/main/changelog#readme. And it should be a feature rather than a bugfix in the file name.

```suggestion
Updated exceptions to save and restore more properties
during the pickle/serialization process.
```